src/source/Ice/IceAgent.c

```diff
@@ -1324,7 +1335,9 @@ STATUS iceAgentInitSrflxCandidate(PIceAgent pIceAgent)
         pCurNode = pCurNode->pNext;
         pCandidate = (PIceCandidate) data;
 
-        if (pCandidate->iceCandidateType == ICE_CANDIDATE_TYPE_HOST) {
+        // TODO: Stop skipping IPv6. Stun serialization and deserialization needs to be implemented properly first.
```

Comment:
Add a GitHub issue tracking this work

Reply:
https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/issues/394